README-it.md

```diff
@@ -0,0 +1,325 @@
+<p align="center">
+    <img alt="oss image" src="./imgs/zoss-logo.svg" height="200px" width="200px">
+</p>
+
+<p align="center">
+  <a href="https://twitter.com/intent/tweet?text=How%20to%20promote%20your%20open-source%20projects%20@ZenikaOSS&url=https://github.com/zenika-open-source/open-source-promotion-cheat-sheet&hashtags=OpenSource,CheatSheet">
+    <img alt="tweet" src="https://img.shields.io/twitter/url/https/twitter?label=Share%20on%20twitter&style=social" target="_blank" />
+  </a>
+</p>
+
+<h1 align="center">Promuovere progetti open source</h1>
+
+> In questi appunti sono riassunti alcuni passaggi importanti da seguire per poter promuovere efficacemente il vostro progetto open source. Potete cliccare un elemento per espandere l'argomento ed avere più informazioni.
+
+List of available languages:
+
+- [Inglese](./README.md)
+- [العربية](./README-ar.md)
+- [Catalano](./README-ca.md)
+- [Tedesco](./README-de.md)
+- [Spagnolo](./README-es.md)
+- [پارسی](./README-fa.md)
+- [Francese](./README-fr.md)
+- [Italiano](./README-it.md)
+- [日本語](./README-jp.md)
+- [Portoghese](./README-pt.md)
+- [Russo](./README-ru.md)
+- [සිංහල](./README-si.md)
+- [Turco](./README-tr.md)
+- [简体中文](./README-zh-cn.md)
+- [繁體中文](./README-zh-tw.md)
```

Comment:
Usually we leave each language name written in its own language, so `Inglese` -> `English` etc.

Reply:
Looks good! I didn't think about that! 
Pushed the fixed list.